feeder/feeder-rl/config.json

```diff
@@ -2,11 +2,14 @@
   "tls": "yes",
   "cert_path_name": "certificate.pem",
   "name_spaces": ["vss"],
-  "broker_url": "personal-l41hw7zrf4-demo-uo7acw3qiq-ez.a.run.app",
+  "broker_url": "w3c-viss-sdv-vccdefault-sglnqbpwoa-ez.a.run.app",
   "port":"443",
   "client_id" : "w3c-vissv2-server",
-  "api_key": "4CA32399-88237AAA-64EB4EE4-541E1D8F",
+  "api_key": "91808E78-38BC0730-1B9FDCEF-47AEFE84",
```

Comment:
That should be an environment variable. Secrets should for sure not be in a public github repo

Reply:
We need a major overview of configuration files and env variables. It's scattered all over the place. This file should not have been version controlled, just a template.

Reply:
Good!